### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by bosgeluk** [lovell/sharp#1999]
I've got sharp running on Lambda. But somehow the local install got corrupt, really no idea, tried to re-install at least 30 times, reset npm cache, tried yarn, removed node_modules etc..

https://sharp.pixelplumbing.com/en/stable/install/
**:: YES**

Have you ensured the platform and version of Node.js used for `npm install` is the same as the platform and version of Node.js used at runtime?
**:: YES
nvm current >> v10.15.3**

If you are installing as a `root` or `sudo` user, have you tried with the `npm install --unsafe-perm` flag?
**:: NO**

What is the complete output of running `npm install --verbose sharp?
**::Attached**
[node-install-verbose-sharp.txt](https://github.com/lovell/sharp/files/3927531/node-install-verbose-sharp.txt)

Have you checked this output for useful error messages?
**:: YES found none**

What is the output of running `npx envinfo --binaries --languages --system --utilities`?
**::Attached**
[npx.txt](https://github.com/lovell/sharp/files/3927538/npx.txt)

**Comment by lovell** [comment]
Sorry, it's unclear what the problem is. The installation log shows no errors. Are you able to explain more about what "local install got corrupt" means?

**Comment by bosgeluk** [comment]
At first Sharp didn't work, then I realized I had node v8 running, I installed v10, activated it with NVM and then sharp worked fine (although at first I used yarn and vaguely I remember that that didn't work out and that I switched to npm). I even setup a second project (inside a folder of the working version) to create an aws lambda layer. I've deployed that layer (serverless) and together with some lambda functions that runs sharp fine on lambda. I had to remove node_modules (plus yarn.lock) for some other reasons and ran yarn install. Sharp installation problem. I remembered the yarn issue, removed node_modules (+yarn.lock - paranoia mode) and ran npm install. Sharp problem. Removed node_modules/sharp and ran npm install. Sharp problem. Retried delete node_modules, package-lock.json and read the output. Npm used a cache from ~/.npm so I deleted the npm cache, node_modules entirely, the package-lock.json file.... Same problem. I checked the node version npm uses ( as suggested in this issue template ) that was node v8. Got it (I thought). Same problem. Deleted npm cache, node_modules, package-lock file... Problem persist.... Tommorow I'm gonna start a fresh project to see if there is some conflict or... I haven't got a clue, maybe I should have rebooted somewhere... To be continued... Cheers
